ignite/distributed/utils.py

```diff
@@ -361,13 +361,20 @@ def all_gather(tensor: Union[torch.Tensor, float, str]) -> Union[torch.Tensor, f
     return _model.all_gather(tensor)
 
 
-def broadcast(tensor: Union[torch.Tensor, float, str], src: int = 0) -> Union[torch.Tensor, float, str]:
+def broadcast(
+    tensor: Union[torch.Tensor, float, str, None], src: int = 0, use_none: bool = False
```

Comment:
`use_none` is not very expressive. Why not something like `allow_none` or `allow_undefined_tgt_tensors` or `safe_to_xxx` ? That is purely a matter of taste !!